README.md

```diff
@@ -1,2 +1,5 @@
-Hello!
+# FIWARE Plattform deployment
```

Comment:
typos:

- s/Plattform/Platform/  
- s/deployment/Deployment/

;)

Reply:
fixed in #e77173